load-env.sh

```
@@ -0,0 +1,21 @@
+#!/bin/bash
+# Load environment variables from .env file
+
+if [ -f .env ]; then
+  # Export each non-empty line that doesn't start with #
+  export $(grep -v '^#' .env | xargs)
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)